src/basic/log.c

```diff
@@ -1071,17 +1068,16 @@ int log_struct_iovec_internal(
                         return -ERRNO_VALUE(error);
         }
 
-        for (i = 0; i < n_input_iovec; i++)
```

Comment:
This line change should be in the previous commit, no? Otherwise the previous one won't compile since the function-level `size_t i` was removed?

Reply:
Indeed, something went wrong with `git add -p` here.